### PR TITLE
schema: close files

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: schema/fs.go [hunk 1]
```diff
@@ -169,6 +169,7 @@ func _escFSByte(useLocal bool, name string) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
+		defer f.Close()
```
Comment (jonboulle) [inline schema/fs.go]: This is a generated file (`make schema-fs`), per https://github.com/mjibson/esc/pull/27#issuecomment-226769594

Comment (runcom) [inline schema/fs.go]: noticed will drop

Comment (runcom) [inline schema/fs.go]: dropped

**Comment by jonboulle** [comment]
LGTM

**Comment by philips** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/opencontainers/image-spec/pull-request/157/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=opencontainers/image-spec)

**Pull request merged**